python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/dlt/asset_decorator.py

```diff
@@ -81,32 +132,24 @@ def github_reactions_dagster_assets(context: AssetExecutionContext, dlt: DltDags
                 yield from dlt.run(context=context)
 
     """
-    dlt_dagster_translator = (
-        check.opt_inst_param(dlt_dagster_translator, "dlt_dagster_translator", DagsterDltTranslator)
-        or DagsterDltTranslator()
+    dlt_dagster_translator_legacy = check.opt_inst_param(
+        dlt_dagster_translator, "dlt_dagster_translator", DagsterDltTranslator
+    )
+    dagster_dlt_translator = check.opt_inst_param(
+        dagster_dlt_translator, "dagster_dlt_translator", DagsterDltTranslator
+    )
+    dagster_dlt_translator = (
+        dagster_dlt_translator or dlt_dagster_translator_legacy or DagsterDltTranslator()
     )
```

Comment:
prob could just stick it in one line

```suggestion
    dagster_dlt_translator = check.inst_param(
        dagster_dlt_translator or dlt_dagster_translator or DagsterDltTranslator(), "dagster_dlt_translator", DagsterDltTranslator
    )
```

Reply:
Good call. :D